sherpa/plot/pylab_backend.py

```diff
@@ -72,30 +70,71 @@ def _choose(test, iftrue, iffalse=None):
 def clear_window():
     pylab.clf()
 
+
 def set_window_redraw(redraw):
     if redraw:
         pylab.draw()
 
 
+def setup_axes(overplot, clearwindow):
+    """Return the axes object, creating it if necessary.
+
+    Parameters
+    ----------
+    overplot : bool
+    clearwindow : bool
+
+    Returns
+    -------
+    axis
+        The matplotlib axes object.
+    """
+
+    # Do we need to clear the window?
+    if not overplot and clearwindow:
+        clear_window()
+
+    return pylab.gca()
+
+
+def setup_plot(axes, title, xlabel, ylabel, xlog=False, ylog=False):
+    """Basic plot setup.
+
+    Parameters
+    ----------
+    axes
+        The plot axes (output of setup_axes).
+    title, xlabel, ylabel : str or None
+        The plot, x-axis, and y-axis titles. They are skipped if
+        the empty string or None.
+    xlog , ylog : bool
+        Should the scale be logarithmic (True) or linear (False)?
+
+    """
+
+    axes.set_xscale('log' if xlog else 'linear')
+    axes.set_yscale('log' if ylog else 'linear')
+
+    if title:
+        axes.set_title(title)
+    if xlabel:
+        axes.set_xlabel(xlabel)
+    if ylabel:
+        axes.set_ylabel(ylabel)
+
+
 def point(x, y, overplot=True, clearwindow=False,
           symbol=None,
           color=None):
 
-    if overplot:
-        axes = pylab.gca()
-    else:
-        if clearwindow:
-            clear_window()
-        axes = pylab.gca()
+    axes = setup_axes(overplot, clearwindow)
 
     if color is None:
-        str = '%s'%(symbol)
+        str = '%s' % (symbol)
```

Comment:
I decided I was going to leave the other bits of code alone (other than things that made pycodestyle complain), but perhaps it's worth cleaning this code up.

For instance, is there any benefit to using `pylab` rather than `matplotlib.pyplot` (other than a smaller import)?

Reply:
I'm not aware of any benefit of importing pylab.

Reply:
This has been changed now.